e2e/js_image_oci/pull.bzl

```diff
@@ -0,0 +1,30 @@
+"a repository rule fetching container images using crane"
```

Comment:
Should users implement their own oci_pull rule like this? 
I'd expect at least a TODO about making a supported upstream API to do that.

Reply:
filed https://github.com/bazel-contrib/rules_oci/issues/63